sentinel-cluster/sentinel-cluster-client-default/src/test/java/com/alibaba/csp/sentinel/cluster/client/codec/data/FlowRequestDataWriterTest.java

```diff
@@ -0,0 +1,62 @@
+package com.alibaba.csp.sentinel.cluster.client.codec.data;
+
+import com.alibaba.csp.sentinel.cluster.request.data.FlowRequestData;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Test cases for {@link FlowRequestDataWriter}
+ *
+ * @author nick-tan
+ * @version 1.0
+ * @date 2019/10/19
+ */
+public class FlowRequestDataWriterTest {
+
+	/**
+	 * normal
+	 */
+	@Test
+	public void writeTo() {
+		FlowRequestDataWriter writer = new FlowRequestDataWriter();
+
```

Comment:
We use 4 spaces as the indent characters, rather than tab. Please follow the Alibaba Java coding guidelines.

Reply:
It's caused by IDE's Smart Tabs.I will close it and reformat code.